tests/sentry/incidents/test_logic.py

```diff
@@ -1425,6 +1534,119 @@ def test_pagerduty_not_existing(self):
                 integration_id=integration.id,
             )
 
+    @responses.activate
+    def test_discord(self):
+        base_url: str = "https://discord.com/api/v10"
+        channel_id = "channel-id"
+        guild_id = "example-discord-server"
+        guild_name = "Server Name"
+        integration = Integration.objects.create(
```

Comment:
Since you have to define this several times maybe you can just put it as `self.discord_integration` in one of the base classes.

Reply:
Only this test is needed, so I don't think I need to define it in the base classes